manager/assets/modext/widgets/modx.panel.welcome.js

```diff
@@ -51,17 +51,17 @@ Ext.extend(MODx.panel.Welcome, MODx.Panel,{
                     success: {
                         fn: function(response) {
                             if (response.success) {
-                                container.update(response.object.html);
+                                container.update(MODx.util.safeHtml(response.object.html, '<h1><h2><h3><h4><span><div><ul><li><a><br><i><em><b><strong>'));
```

Comment:
Probably should add a few more tags to this as well: `<p><ol><dl><dd><dt>` and maybe `<img>`

Reply:
Done.